packages/http/src/router.ts

```diff
@@ -571,11 +573,9 @@ export class Router {
 
         let parseBodyLoading = '';
         if (enableParseBody) {
-            const parseBodyVar = compiler.reserveVariable('parseBody', parseBody);
-            const formVar = compiler.reserveVariable('form', this.form);
+            const parseBodyVar = compiler.reserveVariable('parseBody', this.parseBody.bind(this));
             parseBodyLoading = `
-            const bodyParsed = (await ${parseBodyVar}(${formVar}, request, uploadedFiles));
-            const bodyFields = {...bodyParsed.fields, ...bodyParsed.files};`;
+            const bodyFields = (await ${parseBodyVar}(request, uploadedFiles));`;
```

Comment:
no need for two formats of this data. If the distinction is ever needed, we can bring back the other format with separate files representation